core-bundle/src/Cron/Cron.php

```diff
@@ -87,7 +111,7 @@ public function run(string $scope): void
                 // Check if the cron should be run
                 $expression = CronExpression::factory($interval);
 
-                if (null !== $lastRunDate && $now < $expression->getNextRunDate($lastRunDate)) {
+                if (!$force && null !== $lastRunDate && $now < $expression->getNextRunDate($lastRunDate)) {
```

Comment:
Using the `--force` flag should ideally get a unit test, so that we don't mess it up in the future. :see_no_evil:

Reply:
Unit test added.